### DISCUSSION TITLE
Dataset source idea

### DISCUSSION THREAD

**Discussion by komorra** [PKU-YuanGroup/Open-Sora-Plan#71] [Ideas]
Maybe there is an option to use camera flyover sequences from Gaussian Splatting scenes? What I mean is to use already scanned Gaussian Splatting environments (which offer quite a realistic look) in some renderer (maybe even Unity) to record various camera takes. Here are some links: 

1. https://poly.cam/tools/gaussian-splatting - for scenes download
2. https://github.com/aras-p/UnityGaussianSplatting - for real-time rendering inside Unity

Let me know what you think

**Comment by LinB203** [comment]
Great, we are currently collecting data in this area. Has been added to our todo list.

**Comment by 88572** [comment]
I think a good idea would be to combine some available datasets with content from the open source community.

**Comment by Njasa2k** [comment]
New Dataset: https://huggingface.co/datasets/nkp37/OpenVid-1M
